src/components/MessagesList/MessagesList.vue

```diff
@@ -670,9 +670,10 @@ export default {
 						}
 						this.$store.dispatch('setFirstKnownMessageId', { token, id: startingMessageId })
 						this.$store.dispatch('setLastKnownMessageId', { token, id: startingMessageId })
-
+						// If -2, it means it's the beginning of the chat, so we need to get the context 
+						// using 0
 						// Get chat messages before last read message and after it
-						await this.getMessageContext(token, startingMessageId)
+						await this.getMessageContext(token, startingMessageId !== -2 ? startingMessageId : 0)
```

Comment:
https://github.com/nextcloud/spreed/blob/3071d4dab62ebc974b296197bc68e84a0ed5100f/src/store/messagesStore.js#L982

should respect -2